core/packages/test/scripts/set-env.sh

```diff
@@ -127,3 +126,14 @@ check_tool() {
         exit
     fi
 }
+
+wait_contract_deployed() {
+    ready=""
```

Comment:
To ensure the variable isn't set in the global scope, use [local](https://tldp.org/LDP/abs/html/localvar.html):

```suggestion
    local ready=""
```